free-programming-interactive-tutorials-en.md

```diff
@@ -295,6 +308,12 @@
 * [SQLBolt](http://sqlbolt.com)
 
 
+### PostgresSQL
+
+* [PostgreSQL Tutorial](https://www.postgresqltutorial.com/)
+* [Learn PostgreSQL](https://www.tutorialspoint.com/postgresql/index.htm)
```

Comment:
we have been adding tutorialspoint content to the books and courses list. see #4573